### PR TITLE
Replaced Echo dependency from the core integration with a hook

### PR DESCRIPTION
**Pull request by Michal-Chatlas1221**
As far as I can tell this is the minimal set of changes required for the extension to work, and let us override notification. There are more Echo references, and echo-specific hook handlers which I've decided to leave alone, as they seem to be dead code in our context, and removing them would introduce too much change from the original extension to remain easy to upgrade

### CODE REVIEW

**Comment by emkarcinos** [comment]
Have you tried running Phan on the extension's code? I was wondering if we could switch it on with possibly very little effort.

**Comment by Michal-Chatlas1221** [comment]
> Have you tried running Phan on the extension's code? I was wondering if we could switch it on with possibly very little effort.

Yep I have, and it has a hard time with all the Echo referenced in dead parts of extension. We would need to disable a lot of the checks or go and change a bunch of extension code. 

In general, I think that we were wrong to even put that into ACs in the first place. It's not a Gamepedia extension that we treat like third party, even though it's technically owned by us. In this case we probably want to keep it as close to third party original as we can, and trying to run our PR checks on this doesn't give us much value

**Review by mszabo-wikia — COMMENTED** [review]
Let's try to get Phan running. If we checkout Echo as well, I believe it should discover its classes and work.

**Comment by Michal-Chatlas1221** [comment]
> Let's try to get Phan running. If we checkout Echo as well, I believe it should discover its classes and work.

The branch is currently checking out Echo and Flow, but I don't think the classes are being loaded

File: .github/workflows/php-pr-checks.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        uses: actions/checkout@v3
+        with:
+          repository: wikimedia/mediawiki-extensions-Echo
+          path: mediawiki/extensions/Echo
+
+      - name: Checkout Flow (implicit dependency) extension
+        uses: actions/checkout@v3
+        with:
+          repository: wikimedia/mediawiki-extensions-Flow
+          path: mediawiki/extensions/Echo
```
Comment (mszabo-wikia) [inline .github/workflows/php-pr-checks.yml]: this path is not correct :)

Comment (Michal-Chatlas1221) [reply .github/workflows/php-pr-checks.yml]: image

**Review by kasperekt — APPROVED** [review]

**Pull request merged**